client/ts/package.json

```diff
@@ -19,40 +21,30 @@
     "compile-src:windows": "./compile-proto.bat",
     "docgen": "typedoc",
     "prepackage": "yarn compile-src && yarn build",
-    "package": "yarn --cwd out install && yarn --cwd out cache clean && yarn --cwd out pack -f omega-edit-v$npm_package_version.tgz",
-    "install-omega-edit": "yarn add file://$INIT_CWD/omega-edit-v$npm_package_version.tgz",
-    "pretest": "yarn package && yarn install-omega-edit",
-    "test": "mocha --exit --timeout 50000 --slow 10000 --require ts-node/register --require tests/fixtures.ts --exclude ./tests/specs/server.spec.ts ./tests/specs/*.spec.ts && mocha --exit --timeout 10000 --slow 1000 --require ts-node/register ./tests/specs/server.spec.ts || (yarn remove omega-edit && exit 1)",
-    "lint": "prettier --check webpack.config.js src tests && eslint .",
-    "posttest": "yarn remove omega-edit",
-    "lint:fix": "prettier --write webpack.config.js src tests && eslint --fix ."
+    "package": "yarn --cwd out cache clean && yarn --cwd out pack --install-if-needed -f omega-edit-node-client-v$npm_package_version.tgz",
+    "install-client-local": "yarn add file://$INIT_CWD/omega-edit-node-client-v$npm_package_version.tgz",
+    "pretest": "yarn package && yarn install-client-local",
+    "test": "mocha --exit --timeout 50000 --slow 10000 --require ts-node/register --require tests/fixtures.ts --exclude ./tests/specs/server.spec.ts ./tests/specs/*.spec.ts && mocha --exit --timeout 10000 --slow 1000 --require ts-node/register ./tests/specs/server.spec.ts || (yarn posttest && exit 1)",
+    "posttest": "yarn remove @omega-edit/client",
+    "lint": "prettier --check package.json webpack.config.js src tests && yarn eslint .",
+    "lint:fix": "prettier --write package.json webpack.config.js src tests && yarn eslint --fix ."
```

Comment:
I don't think we need to say `yarn` before `eslint`.

Reply:
I think you are right will update that.